### ISSUE TITLE
List of pipeline and quality related fields

### ISSUE THREAD

**Issue by dvenprasad** [AlexsLemonade/refinebio#156]
To decide on how to present the pipeline and quality information on the UI, I have a few questions/clarifications. Here are the list of fields/information which are important to be displayed:
- Submitter provided protocol (if available and applicable) 
- Steps in the Pipeline + what is being done to the data at each step
- Quality information 

@jaclyn-taroni @Miserlou @kurtwheeler 
Questions:
- What does the pipeline information we store look like?
- Pipelines are for a per file, per sample basis. What is the naming convention for a pipeline? Is the name human-readable? 
- The system should be able to identify whether subset of samples or all samples have been processed through the same pipeline and display pipeline information on a per group/ experiment level, rather than repeat the same pipeline information for multiple samples. Is that possible? Are there any constraints or concerns I should be aware of? 
- ~~On an experiment level, is it possible to have versioning? For example, Version A of an experiment can mean Pipeline X has been applied to some of samples and Y has been applied to some others. Version B can mean Pipeline Y and Pipeline Z have been applied to different subsets of samples. This will make it easier represent experiments where multiple pipelines have been applied to samples?~~ (Not in scope for Keytar Kurt)
- ~~What does the output of quality check look like? Is it a file, JSON, etc.?~~ (Using MultiQC for this)

**Comment by jaclyn-taroni** [comment]
Let's use the quality information listed in #131 as an example, specifically:

> We are specifically interested in exposing `compatible_fragment_ratio`, `strand_mapping_bias` from `lib_format_counts.json` and `percent_mapped`, `library_types` from `aux_info/meta_info.json`

**Comment by Miserlou** [comment]
Ah, okay. Yes, we do get that information. We don't get anything produced by QC-specific tools, ex `salmontools`.

**Comment by jaclyn-taroni** [comment]
@dvenprasad can you take a look at this and give us an update about what would be required to close this/when that might happen?

**Comment by jaclyn-taroni** [comment]
Depends on: #170

**Comment by jaclyn-taroni** [comment]
#170 has been closed, so I will close this as well.
